packages/builder/src/index.ts

```diff
@@ -94,8 +95,15 @@ async function run(
     context.logger.info('All files pass linting.\n');
   }
 
+  const tooManyWarnings = maxWarnings >= 0 && totalWarnings > maxWarnings;
+  if (tooManyWarnings && printInfo) {
+    context.logger.error(
+      `Found ${totalWarnings} warnings, which exceeds your configured limit (${options.maxWarnings}). Either fix some of them or increase the value of the maxWarnings option.`,
```

Comment:
Hard to explain, but to me as a native English speaker the following inversion of the second sentence makes it come across less bluntly than your current text:

```suggestion
      `Found ${totalWarnings} warnings, which exceeds your configured limit (${options.maxWarnings}). Either increase your maxWarnings limit or fix some of the lint warnings.`,
```

Reply:
(You may well be a native English speaker too, I'm referencing that fact about me to justify why it's more of a feeling than anything else :) )